src/tinklj/daead.clj

```diff
@@ -0,0 +1,15 @@
+(ns tinklj.daead
+  (:require [tinklj.keys.keyset-handle :as keyset-handles])
+  (:import (java.security Security)))
+
+(Security/setProperty "crypto.policy" "unlimited")
```

Comment:
what does this do?

Reply:
Oh, this allows unlimited encryption to be used: https://stackoverflow.com/questions/24907530/java-security-invalidkeyexception-illegal-key-size-or-default-parameters-in-and

(Basically why everything was broken for so long on my branch)

Reply:
Turns out it wasn't needed, removed.